newIDE/app/src/PropertiesEditor/MeasurementUnitDocumentation.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import { Column } from '../UI/Grid';
+import Text from '../UI/Text';
+
+// Strings are used instead of gdMeasurementUnit
+// because the C++ instance would be dead when the component is rendered.
+type Props = {|
+  label: string,
+  description: string,
+  elementsWithWords: string,
+|};
+
+export default function MeasurementUnitDocumentation({
+  label,
+  description,
+  elementsWithWords,
+}: Props) {
+  return (
+    <Column noMargin>
+      <Text size="sub-title">{label}</Text>
+      {description && <Text size="body">{description}</Text>}
+      {elementsWithWords && (
+        <Text size="body">
+          <i>
+            <Trans>{`in ${elementsWithWords}`}</Trans>
```

Comment:
The name is not great. I think it won't help translators much, but for what there is is to translate it may be ok.

Reply:
I'm not sure the variable name appears in crowding, they will see `in {0}`